bucket/dopamine.json

```diff
@@ -5,6 +5,7 @@
     "license": "GPL-3.0-only",
     "url": "https://www.digimezzo.com/content/software/dopamine/Dopamine%202.0.9%20(Release)%20-%20Portable.zip",
     "hash": "fd5ec9aa1c6f9a2a21244a65f9d77ef20614422c9ce6df75614ba8edb09c8b08",
+    "post_install": "if (!(Test-Path \"$persist_dir\\Dopamine\\Settings.xml\")) { New-Item \"$dir\\Dopamine\\Settings.xml\" | Out-Null }",
```

Comment:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\Dopamine\\Settings.xml\")) { New-Item \"$dir\\Dopamine\\Settings.xml\" | Out-Null }",
```

Reply:
Umm isn't it got checked after installation?

Reply:
Hard link is created before post_install. So we need to create the file before the hard link creation.